cmd/state-custom-installer/runUnInstall.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/ActiveState/cli/internal/config"
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/fileutils"
+
+	"github.com/ActiveState/cli/internal/osutils"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/internal/subshell"
+	"github.com/ActiveState/cli/internal/subshell/sscommon"
+)
+
+// NOTE: From: internal/runners/clean/uninstall.go
+
+func runUnInstall(out output.Outputer, params *Params) error {
+	// This is the deploy id constant: DeployID RcIdentification = RcIdentification{
+	installToDir := params.path
+	// Should look for license file to see if it exists before we delete the install directory
+	uninstallFileCheck := filepath.Join(installToDir, "LICENSE.txt")
```

Comment:
This isn't much of a verification imo, `LICENSE.txt` is a very common filename and does not uniquely identify our runtime installation dir. Maybe also check if it contains 'ActiveState' or something?